docs/application/web/api/2.3.2/device_api/wearable/tizen/bluetooth.html

```diff
@@ -4085,6 +4085,9 @@ <h4>Methods</h4>
 <p><span class="privilege">Privilege: </span>
  http://tizen.org/privilege/bluetooth.admin
             </p>
+<p><span class="remark">Remark: </span>
+ Establishing a connection can make device's Bluetooth stack busy. Some operations, for example <a href="#BluetoothLEAdapter::stopScan">BluetoothLEAdapter::stopScan()</a>, may fail before <em>connect()</em>'s <em>successCallback()</em> or <em>errorCallback()</em> is called.
+            </p>
```

Comment:
Can we modify this slightly? 
What is your opinion, & also please check for technical accuracy if you accept the modification. 
Global: 
<p><span class="remark">Remark: </span>
 Establishing a connection can make device's bluetooth stack busy. With the result, some of the operations, such as <a href="#BluetoothLEAdapter::stopScan">BluetoothLEAdapter::stopScan()</a>, can fail even before calling <em>connect()'s</em> callbacks such as<em>  successCallback()</em> or <em>errorCallback()</em>.
           </p>

Reply:
Hello, unfortunately sentence changed the meaning we need. Please check below version:

Remark: Establishing a connection can make device's bluetooth stack busy. With the result, some of the operations, such as BluetoothLEAdapter::stopScan(), can fail until connect()'s callback such as successCallback() or errorCallback() is triggered.

Reply:
@pkosko that is the reason , I asked you to verify, Thnks a lot! 

Remark: Establishing a connection can make device's bluetooth stack busy. With the result, some of the operations, such as BluetoothLEAdapter::stopScan() can fail until connect()'s callback such as successCallback() or errorCallback() is triggered.

Looks fine !

Reply:
@nawab-samsung  I've uploaded changed description to all files. In case of merging please use 'Squash and merge' strategy to make PR single commit on master branch.